README.md

```diff
@@ -21,7 +21,7 @@ Self-contained C++ interface for the [MeshCat visualizer](https://github.com/rde
 
 ## 🏗️ Dependencies
 
-**meshcat-cpp** is a self-contained library. Most dependencies can be cloned at compile time and they are statically linked to the library. The only dependencies you need is a sufficiently recent C++ compiler (full support to C++17), `cmake`, `openssl`, `zlib`, `libuv`, `boost` and `pkg-config`.
+**meshcat-cpp** is a self-contained library. Most dependencies can be cloned at compile time and they are statically linked to the library. The only dependencies you need is a sufficiently recent C++ compiler (full support to C++20), `cmake`, `openssl`, `zlib`, `libuv`, `boost` and `pkg-config`.
```

Comment:
Strictly speaking we do not need full support for C++20 (Ubuntu 20.04 works with partial support), but we can also keep the phrase like that to keep it simple.